Encodings.md

```diff
@@ -335,14 +335,15 @@ Note that, even for FIXED_LEN_BYTE_ARRAY, all lengths are encoded despite the re
 
 ### Byte Stream Split: (BYTE_STREAM_SPLIT = 9)
 
-Supported Types: FLOAT, DOUBLE
+Supported Types: INT32, INT64, FLOAT, DOUBLE, FIXED_LEN_BYTE_ARRAY
```

Comment:
What about still placing FLOAT and DOUBLE in the beginning?

Reply:
Or add some comment to let readers know the format versions they have been added?

Reply:
I think it would be a good idea in general to have tables of version information for Parquet features, but we don't seem to do that in this file.

Reply:
Done, I moved FLOAT and DOUBLE to the beginning, and I added a note about versions in the Thrift document (where this seems to be mentioned for other features).